opennms-container/minion/container-fs/entrypoint.sh

```diff
@@ -152,7 +152,7 @@ applyOverlayConfig() {
   # Overlay etc specific config
   if [ -d "${MINION_OVERLAY_ETC}" ] && [ -n "$(ls -A ${MINION_OVERLAY_ETC})" ]; then
     echo "Apply custom etc configuration from ${MINION_OVERLAY_ETC}."
-    cp -Lr ${MINION_OVERLAY_ETC}/* ${MINION_HOME}/etc || exit ${E_INIT_CONFIG}
+    rsync -r --out-format="%n %C" ${MINION_OVERLAY_ETC}/* ${MINION_HOME}/etc || exit ${E_INIT_CONFIG}
```

Comment:
Does this not technically change the behavior? Do you need to add `-L` to the `rsync` as well?

Reply:
Also anything that syncs into a directory should have `/` at the end since `cp` has somewhat different semantics. So it should be:

```bash
rsync -r -L --out-format="%n %C" ${MINION_OVERLAY_ETC}/* ${MINION_HOME}/etc/ || exit ${E_INIT_CONFIG}
```

...if we're trying to not change behavior.

(I didn't go look at the other rsync's but I bet that's the case for a lot of these.

It might just happen to work anyway this time, but I strongly recommend making sure your slashes are appropriate because it's easy for it to go wrong with rsync.

Reply:
> Does this not technically change the behavior? Do you need to add `-L` to the `rsync` as well?

I thought `-L` in rsync did something different from `-L` in cp, but was a misunderstanding on my end and reading the **wrong** man page for cp (I was looking at macOS, which requires that `-R` be passed for `-L` to do anything, but the one that runs in the container is GNU coreutils, which doesn't seem to have such a requirement).

Reading the *detailed* docs for both, it sounds like they are equivalent, so the `-L` should be there, so I'll add that back.

GNU coreutils:
> `-L`, `--dereference`
> Follow symbolic links when copying from them. With this option, cp cannot create a symbolic link. For example, a symlink (to regular file) in the source tree will be copied to a regular file in the destination tree.

-- https://www.gnu.org/software/coreutils/manual/html_node/cp-invocation.html

rsync:
> `-L`, `--copy-links`
> When symlinks are encountered, the item that they point to (the referent) is copied, rather than the symlink. In older versions of rsync, this option also had the side-effect of telling the receiving side to follow symlinks, such as symlinks to directories. In a modern rsync such as this one, you'll need to specify --keep-dirlinks (-K) to get this extra behavior. The only exception is when sending files to an rsync that is too old to understand -K -- in that case, the -L option will still have the side-effect of -K on that older receiving rsync.

-- https://linux.die.net/man/1/rsync